### DISCUSSION TITLE
Source code for openpnm.algorithms.metrics.RelativePermeability algorithm

### DISCUSSION THREAD

**Discussion by minzhangut** [PMEAL/OpenPNM#2439] [Q&A]
Hello Professor @jgostick,

I am wondering whether you could share the source code for openpnm.algorithms.metrics.RelativePermeability algorithm. I did not see it  in the Module Reference section.

Thanks and best regards!

**Comment by jgostick** [comment]
Hi @minzhangut 
as you know, we are renovating V2 to become V3, so lots of code is moving around or temporarily removed.  However, you can always find it by selecting the "branch" from the dropdown box on the top left of the code page.  If you select 'release' you'll see the source code corresponding to V2.8.2, which is the latest official release.  Once there you can navigate to /openpnm/metrics/RelativePermeability and see the code you need.
